cg/cli/upload/clinical_delivery.py

```diff
@@ -67,12 +67,14 @@ def upload_clinical_delivery(context: click.Context, case_id: str, dry_run: bool
         {"jobs": [str(job_id)]}, config_path=rsync_api.trailblazer_config_path
     )
     analysis_name: str = f"{case_id}_rsync" if is_complete_delivery else f"{case_id}_partial"
+    order_id: int = case.latest_order.id
```

Comment:
How does this work when a reanalysis is requested? I think they create a new order most times, so it probably will work fine

Reply:
Good question!

Reply:
Double-checked with @beatrizsavinhas who confirmed that re-analyses are placed in new orders!